plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -199,7 +199,7 @@ public GlueHiveMetastore(
                 config.getPartitionSegments(),
                 config.isAssumeCanonicalPartitionKeys(),
                 visibleTableKinds,
-                taskWrapping(newThreadPerTaskExecutor(Thread.ofVirtual().name("glue-", 0L).factory())));
+                newFixedThreadPool(config.getThreads(), Thread.ofPlatform().name("glue-", 0L).factory()));
```

Comment:
> Switch Glue metastore to platform threads

why this particular change?

(i like it because it makes it possible to detect thread leaks. the API used by https://github.com/trinodb/trino/pull/21470 doesn't report virtual threads currently, but i assume this is not the reason)